frb_example/pure_dart/dart/lib/main.dart

```diff
@@ -655,6 +660,10 @@ void main(List<String> args) async {
       expect(resp.millisecondsSinceEpoch, date.millisecondsSinceEpoch);
       expect(resp.microsecondsSinceEpoch, date.microsecondsSinceEpoch);
     });
+    test('Empty DateTime', () async {
+      final resp = await api.optionalEmptyDatetimeUtc(d: null);
```

Comment:
seems we forget to test boxed_empty_datetime_utc here

Reply:
Added